src/modules/tools/zprobe/ZProbe.cpp

```diff
@@ -390,3 +411,29 @@ float ZProbe::zsteps_to_mm(float steps)
 {
     return steps / Z_STEPS_PER_MM;
 }
+
+void ZProbe::deploy_servo()
+{
+	if(this->servo_pin == NULL) return;
+
+	this->servo_pin->period_ms(20);
+	this->servo_pin->pulsewidth_us(1000 + this->servo_percent * 10);
+
+	if(this->servo_disable) {				//Disable servo to eliminate jitter
+		wait_ms(250);
```

Comment:
You cannot put delays like this in the code. it needs to loop on an on_idle call, you can look at other modules for examples